test/powershell/engine/Module/TestModuleManifest.ps1

```diff
@@ -0,0 +1,59 @@
+Describe "Test-ModuleManifest tests" -tags "CI" {
+
+    AfterEach {
+        if (Test-Path testdrive:/module) {
+            Remove-Item -Recurse -Force testdrive:/module
+        }
+    }
+
+    It "module manifest containing paths with backslashes or forwardslashes are resolved correctly" {
+
+        New-Item -ItemType Directory -Path testdrive:/module
+        New-Item -ItemType Directory -Path testdrive:/module/foo
+        New-Item -ItemType Directory -Path testdrive:/module/bar
+        "" > testdrive:/module/foo/bar.psm1
+        "" > testdrive:/module/bar/foo.psm1
+        $testModulePath = "testdrive:/module/test.psd1"
+        $fileList = "foo\bar.psm1","bar/foo.psm1"
+
+        New-ModuleManifest -NestedModules $fileList -RootModule foo\bar.psm1 -RequiredAssemblies $fileList -Path $testModulePath -TypesToProcess $fileList -FormatsToProcess $fileList -ScriptsToProcess $fileList -FileList $fileList -ModuleList $fileList
+
+        Test-Path $testModulePath | Should Be $true
+
+        Test-ModuleManifest -Path $testModulePath -ErrorAction Stop | Should BeOfType System.Management.Automation.PSModuleInfo
```

Comment:
I used `ErrorAction Stop` here since Test-ModuleManifest writes to the error stream on manifest errors and I thought this was an easy way to ensure no errors being returned.  Otherwise I think I'd have to redirect the error stream and check it and I don't think that's providing additional value.  I can add a comment to clarify my intent.